taskt/Core/Automation/Commands/File Operation Commands/ExtractFilesCommand.cs

```diff
@@ -45,56 +45,50 @@ public class FileExtractionCommand : ScriptCommand
         [XmlAttribute]
         [PropertyDescription("Source File Path / URL")]
         [InputSpecification("Enter or Select the Path / URL to the applicable file.")]
-        [SampleUsage(@"C:\temp\myfile.zip || https://temp.com/myfile.zip || {vFileSourcePath}")]
-        [Remarks("")]
+        [SampleUsage(@"C:\temp\myfile.zip || {ProjectPath}\myfile.zip || https://temp.com/myfile.zip || {vFileSourcePath}")]
+        [Remarks("{ProjectPath} is the directory path of the current project.")]
         [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowFileSelectionHelper)]
         public string v_FilePathOrigin { get; set; }
 
         [XmlAttribute]
         [PropertyDescription("Extracted File(s) Directory Path")]
         [InputSpecification("Enter or Select the Folder Path / URL to move extracted file(s) to.")]
-        [SampleUsage(@"C:\temp\ || {vFilesPath}")]
+        [SampleUsage(@"C:\temp || {vFilesPath}")]
```

Comment:
please add a sample that uses ProjectPath

Reply:
done